### ISSUE TITLE
Plagiarism detector on content

### ISSUE THREAD

**Issue by sydneyitguy** [Steemhunt/web#148]
maybe research on APIs:

https://plagiarismdetector.net/
https://smallseotools.com/plagiarism-checker/
http://plagiarisma.net/
https://www.quetext.com/
http://www.dustball.com/cs/plagiarism.checker/
https://edubirdie.com/plagiarism-checker

https://chrome.google.com/webstore/detail/reveye-reverse-image-sear/keaaclcjhehbbapnphnmpiklalfhelgf?utm_source=chrome-app-launcher-info-dialog : chrome extension : reverse image search
http://www.image-search.org/
https://www.tineye.com/

**Comment by chronocrypto** [comment]
Where Its shows the % of your copied text.
Maybe allow a certain % such as quotes.

**Comment by Folken8** [comment]
For now, quotext seems a good alternative (espoem recommand it) BUT it doesn't have any API

**Comment by chronocrypto** [comment]
Of course, it only works if the words are correct, There a few who on purpose omit a letter or two.
